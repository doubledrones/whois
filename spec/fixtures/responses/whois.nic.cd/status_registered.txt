Domain Name: google.cd
Domain ID: 5758-CD
Updated Date: 2013-04-17T14:40:48.947Z
Creation Date: 2006-09-29T22:00:00.000Z
Registry Expiry Date: 2013-09-30T22:00:00.000Z
Sponsoring Registrar: MARKMONITOR
Sponsoring Registrar IANA ID: 
Domain Status: clientTransferProhibited
Domain Status: clientUpdateProhibited
Domain Status: ok
Domain Status: clientRenewProhibited
Domain Status: clientDeleteProhibited

Registrant ID: 5579-CD

Admin ID: 5579-CD

Billing ID: 5566-CD

Tech ID: 5572-CD
Tech Phone: 1.650253
Tech Phone Ext: 
Tech Fax: 1.650618857
Tech Fax Ext: 
Tech Email: dns-admin@google.com

Name Server: ns1.google.com
Name Server: ns2.google.com
Name Server: ns3.google.com
Name Server: ns4.google.com

DNSSEC: unsigned


TERMS OF USE: You are not authorized to access or query our WHOIS database through the use of electronic processes that are high-volume and automated. This WHOIS database is provided by CDnic as a service to the internet community on behalf of Congo, the Democratic Republic (La Republique Democratique du Congo ? RDC). 

The data is for information purposes only. CDnic does not guarantee its accuracy. By submitting a WHOIS query, you agree to abide by the following terms of use: You agree that you may use this Data only for lawful purposes and that under no circumstances will you use this Data to: (1) allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via e-mail, telephone, or facsimile; or (2) enable high volume, automated, electronic processes that apply to CDnic it's members (or CDnic or member computer systems). The compilation, repackaging, dissemination or other use of this Data is expressly prohibited.


>>> Last update of WHOIS database: 2013-06-30T14:52:15.383Z <<<
