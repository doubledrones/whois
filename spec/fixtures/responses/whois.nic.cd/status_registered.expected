#domain
  should: %s == "google.cd"

#domain_id
  should: %s == "5758-CD"


#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2006-09-29 22:00:00 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-04-17 14:40:48 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-09-30 22:00:00 UTC")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "MARKMONITOR"
  should: %s.organization == nil
  should: %s.url          == nil


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.google.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.google.com"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns3.google.com"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns4.google.com"
